Rocket.Chat/Managers/Model/AuthSettingsManager.swift

```diff
@@ -38,7 +38,7 @@ final class AuthSettingsManager {
                 return
             }
 
-            Realm.executeOnMainThread({ realm in
+            Realm.execute({ realm in
                 let settings = AuthManager.isAuthenticated()?.settings ?? AuthSettings()
                 settings.map(response.result["result"], realm: realm)
                 realm.add(settings, update: true)
```

Comment:
Just crashed here 😦 "not in a write transaction" when changing servers.

Reply:
Looks like this causes very bad problems @rafaelks, can we revert only this part? Can't even connect to servers with this.